packages/pagination/src/usePagination.ts

```diff
@@ -15,7 +15,8 @@ import {
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface IUsePaginationProps<Item> extends IUseSelectionProps<Item> {}
 
-export interface IGetPageProps<Item> extends IGetItemPropsOptions<Item> {
+export interface IGetPageProps<Item> extends Omit<IGetItemPropsOptions<Item>, 'role'> {
+  role?: string | null;
```

Comment:
If `role` is already provided by `IGetItemPropsOptions<T>`, why do we have to redefine it here again? We should be able to inherit correctly from it.

Reply:
It is coming from `HTMLProps` where it is defined as `string | undefined`. In order to remain consistent with other `role` usages throughout containers we redefine as `string | null`. Also seen on line 25 just below.